mindsdb/integrations/handlers/llama_index_handler/llama_index_handler.py

```diff
@@ -269,31 +256,22 @@ def predict(
 
     def _get_service_context(self):
         args = self.model_storage.json_get("args")
-        engine_storage = self.engine_storage
-
-        key = "OPENAI_API_KEY"
-        openai_api_key = get_api_key(
-            key, "llama_index", args["using"], engine_storage, strict=False
-        )
-        if openai_api_key is None:
-            openai_api_key = get_api_key(
-                key.lower(), "llama_index", args["using"], engine_storage, strict=True
-            )
-
+        openai_api_key = self._get_llama_index_api_key(args["using"])
         openai.api_key = openai_api_key  # TODO: shouldn't have to do this! bug?
-        llm_kwargs = {"openai_api_key": openai_api_key}
+        llm_kwargs = {
+            "openai_api_key": openai_api_key,
+            "model": "gpt-3.5-turbo-instruct",
+        }
         if "temperature" in args["using"]:
             llm_kwargs["temperature"] = args["using"]["temperature"]
         if "model_name" in args["using"]:
-            llm_kwargs["model_name"] = args["using"]["model_name"]
+            llm_kwargs["model"] = args["using"]["model_name"]
```

Comment:
Is this key renaming correct? Previously we were passing `model_name` to `langchain.llm.OpenAI`. Does `model` match the expected parameter?

Reply:
yes this was wrong the object takes model as a argument

Reply:
Wait I'll check this once again, and update this correctly

Reply:
We can pass both `model` and `model_name` still changing this to `model_name` as recommended in this [link](https://github.com/langchain-ai/langchain/issues/4331)